docs/pages/docs/core-concepts/remote-caching.mdx

```diff
@@ -10,9 +10,7 @@ import Callout from "../../../components/Callout";
 
 A major key 🔑 to Turborepo's speed is that it is both lazy and efficient—it does the least amount of work possible and it tries to never redo work that's already been done before.
 
-At the moment, your Turborepo caches your tasks on your local filesystem (i.e. "single-player mode," if you will). However, what if there was a way to take advantage of the computational work done by your teammates or your CI (i.e. "co-op multiplayer mode")?
-
-What if there was a way to teleport and share a single cache across machines? Almost like a "Dropbox" for your Turborepo cache.
+By default, Turborepo caches the outputs of your tasks on your machine's local file system. We call this "single-player mode." But what if you could take advantage of the computational work done by your teammates or your CI? What if you could share a single cache across machines? In other words, what if you could turn on "co-op multiplayer mode"? A "Dropbox" for your Turborepo cache.
```

Comment:
This reads a little bit clunky to me - here's another suggestion, but I'm not married to it. 

```suggestion
By default, Turborepo caches the outputs of your tasks on your machine's local file system. We call this "single-player mode." But what if you could take advantage of the computational work done by your teammates or your CI by sharing a single cache across machines? In other words, what if you could turn on "co-op multiplayer mode", or a "Dropbox" for your Turborepo cache?
```